### PR TITLE
[IMP] POS: upgrade the getting started page for 15.0

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
Task ID: 3090486
Do not fw to 16.0 without making other changes

### CODE REVIEW

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 1]
```diff
@@ -23,44 +24,56 @@ Configure payment methods
...
 
 To add a payment method, you first need to create it. Go to :menuselection:`Point of Sale -->
-Configuration --> Payment Methods --> Create`. Set a name and select the **payment terminal** or
-check :guilabel:`Cash` for cash payments.
+Configuration --> Payment Methods --> Create`. Set a name and select the :guilabel:`Journal`. Select
+:guilabel:`Cash` to use this payment method for cash payments, or :guilabel:`Bank` to use it for
+card payments.
+
+Check :guilabel:`Identify Customer` to allow this payment method *exclusively* for identified
+customers. For any unidentified customers to be able to pay, leave the :guilabel:`Identify Customer`
```
Comment (Donapi) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:32]: - Not sure to understand what "identified customers" refer to. 
- I would probably move this part before the "Journal" information as the "Identify Customer" field is above the Journal one.

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 2]
```diff
@@ -23,44 +24,56 @@ Configure payment methods
...
+   when created, add a link to the customeraccount page under advanced_pricing_features
 
 .. image:: getting_started/payment-method.png
    :align: center
    :alt: Creating a new payment method for a POS.
 
 .. note::
-   Credentials are mandatory to use a payment terminal. To learn how to configure the different
-   terminals, check out the following documentation pages:.
+   - Selecting a **bank** journal automatically adds the :guilabel:`Use a Payment Terminal` field in
```
Comment (Donapi) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:43]:
```suggestion
   - Selecting a :guilabel:`Bank` journal automatically adds the :guilabel:`Use a Payment Terminal` field in
```
adding :guilabel: as this option is visible and clickable on screen.

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 3]
```diff
@@ -23,44 +24,56 @@ Configure payment methods
...
+.. _session:
+
 Start a session
 ---------------
 
-From the **POS dashboard**, click :guilabel:`New Session` to start a POS session, or
-:guilabel:`Resume` if the session was already open.
+From the **POS dashboard**, click :guilabel:`New Session` and :guilabel:`Open Session` at the
+:guilabel:`Opening Cash Control` screen to start a POS session, or click :guilabel:`Continue
+Selling` if the session was already opened.
```
Comment (Donapi) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:68]:
```suggestion
From the **POS dashboard**, click :guilabel:`New Session` and at the
:guilabel:`Opening Cash Control` screen click :guilabel:`Open Session` or click :guilabel:`Continue Selling` if the session was already opened.
```

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 4]
```diff
@@ -23,44 +24,56 @@ Configure payment methods
...
 
 .. note::
    Multiple users can be connected to the same session at the same time. However, the session can
-   only be open once on the same browser.
+   only be opened once on the same browser.
 
-Click on products to add them to the cart. To change the **quantity**, click :guilabel:`Qty` and
-enter the number of products using the keypad. To add a **discount** or modify the product
-**price**, click respectively :guilabel:`Disc` or :guilabel:`Price` and enter the amounts.
+Click products to add them to the cart. To change the **quantity**, click :guilabel:`Qty` and enter
```
Comment (Donapi) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:74]:
```suggestion
Click products to add them to the cart. To change the **quantity**, click on the green button :guilabel:`Qty` and enter
```

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 5]
```diff
@@ -80,25 +93,50 @@ Return and refund products
...
+- the number of orders made and the total amount made during the session;
+- the total of all the transactions made during the session grouped by payment method;
+- the expected amounts grouped by payment method.
+
+Before closing this window, count your cash using the calculator icon. Doing so opens a pop-up
+window that computes the total amount in the cash drawer depending on the coins and bills counted
+and added manually. Then, click :guilabel:`Confirm` or :guilabel:`Discard` to close the window. The
+computed amount is set in the :guilabel:`Counted` column, and the :guilabel:`Money Details` are
+specified in the **Notes** section.
+
```
Comment (Donapi) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:126]: I would move the image here as it illustrates the above paragraph 
.. image:: getting_started/closing-control.png
   :align: center
   :alt: How to close a POS session.

**Review by Donapi — CHANGES_REQUESTED** [review]
Great doc! Some changes suggested :)

File: content/applications/sales/point_of_sale/overview/getting_started.rst (hunk 4 above)
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/overview/getting_started.rst:74]: for consistency, I avoid using "click on the"

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 6]
```diff
@@ -23,44 +24,55 @@ Configure payment methods
...
+     terminals, check out the following documentation pages:
 
-   - :doc:`Adyen configuration <../payment/adyen>`
-   - :doc:`Vantiv configuration <../payment/vantiv>`
-   - :doc:`Ingenico configuration <../payment/ingenico>`
-   - :doc:`Six configuration <../payment/six>`
+     - :doc:`Adyen configuration <../payment/adyen>`
+     - :doc:`Vantiv configuration <../payment/vantiv>`
+     - :doc:`Ingenico configuration <../payment/ingenico>`
+     - :doc:`Six configuration <../payment/six>`
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/overview/getting_started.rst:51]: super good catch; worldline is used starting in 15.0, but I totally forgot about that

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 7]
```diff
@@ -23,44 +24,56 @@ Configure payment methods
...
+Once the payment method is created, you can select it in your POS settings. To do so, go to
 :menuselection:`Point of Sale --> Configuration --> Point of Sale` and select a POS for which you
 wish to make the payment method available. Click :guilabel:`Edit` and add the payment method under
 the :guilabel:`Payments` section.
 
-
 POS sessions
 ============
 
+.. _session:
```
Comment (jcs-odoo) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:61]: I suggest something more specific.
It's not a rule at all, but that's why I tend to write a prefix for the doc and then the topic.
```suggestion
.. _pos/sessions:
```

also, if it's about the heading "POS sessions", put it _before_ that heading.
If it is for "start a session" I would then suggest something like
```suggestion
.. _pos/start-session:
```

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 8]
```diff
@@ -23,44 +24,56 @@ Configure payment methods
 -------------------------
 
 To add a payment method, you first need to create it. Go to :menuselection:`Point of Sale -->
-Configuration --> Payment Methods --> Create`. Set a name and select the **payment terminal** or
-check :guilabel:`Cash` for cash payments.
+Configuration --> Payment Methods --> Create`, and set a name. Check :guilabel:`Identify Customer`
+to allow this payment method *exclusively* for registered customers. For any customers to be able to
+pay, leave the :guilabel:`Identify Customer` field unchecked.
```
Comment (jcs-odoo) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:29]:
```suggestion
to allow this payment method *exclusively* for registered customers.
```

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 9]
```diff
@@ -23,44 +24,56 @@ Configure payment methods
...
+.. _session:
+
 Start a session
 ---------------
 
-From the **POS dashboard**, click :guilabel:`New Session` to start a POS session, or
-:guilabel:`Resume` if the session was already open.
+From the **POS dashboard**, click :guilabel:`New Session` and at the :guilabel:`Opening Cash
+Control` screen click :guilabel:`Open Session` to start a POS session, or click :guilabel:`Continue
+Selling` if the session was already opened.
```
Comment (jcs-odoo) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:68]:
```suggestion
Selling` if the session is already opened.
```

File: content/applications/sales/point_of_sale/overview/getting_started.rst [hunk 10]
```diff
@@ -23,44 +24,56 @@ Configure payment methods
...
 
+.. _session:
+
 Start a session
 ---------------
 
-From the **POS dashboard**, click :guilabel:`New Session` to start a POS session, or
-:guilabel:`Resume` if the session was already open.
+From the **POS dashboard**, click :guilabel:`New Session` and at the :guilabel:`Opening Cash
+Control` screen click :guilabel:`Open Session` to start a POS session, or click :guilabel:`Continue
```
Comment (jcs-odoo) [inline content/applications/sales/point_of_sale/overview/getting_started.rst:67]:
```suggestion
Control` screen, click :guilabel:`Open Session` to start a POS session, or click :guilabel:`Continue
```

**Review by jcs-odoo — APPROVED** [review]
Some suggestions, but you can ping doc-review after you've applied them. :)

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**